addOns/commonlib/CHANGELOG.md

```diff
@@ -5,12 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
```

Comment:
I know this is a whitespace change, I did do it on purpose, the extra blank looked bad to me.

Reply:
We should have a formatter.

Reply:
😁